docs/dev-running.md

```diff
@@ -83,9 +83,69 @@ $ fedimint-cli reissue BgAAAAAAAAAgAAAAAAAAAAEAAAAAAAAAwdt...
 }
 ```
 
+### On-Chain Deposit and Withdrawal
+
+#### Deposit
+
+Check you current ecash balance:
+
+```shell
+$ fedimint-cli info
+
+{
+  "denominations_msat": {
+    "1": 2,
+    "1024": 3,
+    "1048576": 3,
+    "128": 3,
+    "131072": 2,
+    "16": 2,
+    "16384": 2,
+    "2": 3,
+    "2048": 2,
+    "2097152": 2,
+    "256": 2,
+    "262144": 2,
+    "32": 2,
+    "32768": 3,
+    "4": 2,
+    "4096": 3,
+    "512": 3,
+    "524288": 3,
+    "64": 2,
+    "65536": 2,
+    "8": 2,
+    "8192": 2
+  },
+  "total_msat": 10000000
+}
+```
+
+To deposit into a Fedimint, you must generate a new deposit address, funds sent to it can later be claimed
```

Comment:
Not sure how to properly communicate this, but funds sent to an onchain address are automatically claimed by the client state machine framework in the background. The user doesn't need to do anything